meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,221 @@
+                    "attributes_ranking_order": 0.8,
+                    "attributes_query_word_order": 0.6363636363636364,
+                    "score": 0.7272727272727273
+                },
+                "exactness": {
+                    "order": 4,
+                    "matchType": "noExactMatch",
+                    "score": 0.3333333333333333
+                }
+            }
+```
+
+⚠️ While the `_rankingScore` is stable in v1.3.0 the `_rankingScoreDetails` are being released as an experimental feature. They are expected to be stabilized in v.1.4.0.
+
+When launching Meilisearch, use the following flag:
+```bash
+meilisearch --experimental-score-details
+```
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/meilisearch/discussions/594).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
+
+Done by @dureuill in #3771
+
+## Define field to search on at search time
+
+v1.3.0 introduces a new search parameter, `attributesToSearchOn`.
+
+`attributesToSearchOn` is a search parameter accepting an array of strings indicating one or more document attributes. Queries using `attributesToSearchOn` will restrict the search to the indicated attributes.
+
+Given the following dataset:
+```json
+{
+  "id": 0,
+  "name": "Our Wives Under the Sea",
+  "genre": ["horror", "scifi"],
+  "synopsis": "A woman returns to her wife transformed after a deep-sea adventure."
+},
+{
+  "id": 1,
+  "name": "A Strange and Stubborn Endurance",
+  "genre": ["adventure"],
+  "synopsis": "A man must overcome trauma and fight off a mysterious assassin."
+}
+```
+And the following query:
+```json
+{
+  "q": "adventure",
+  "attributesToSearchOn": ["genre"]
+}
+```
+Meilisearch will only return document 1.
+
+Both documents contain the word adventure, but `"attributesToSearchOn": ["genre"]` instructs Meilisearch to only consider results found on the genre field.
+
+Done by @ManyTheFish in (#3834)
+
+
+## Search for facet values
+Meilisearch v1.3.0 introduces a new endpoint  `POST /indexes/{index}/facet-search` that allows you to search for facet values. Only fields defined as `filterableAttributes` will be facet searchable.
+
+Done by @Kerollmops in (#3611)
+
+## Sort facet values by count
+By default, all facet values are sorted in ascending alphanumeric order. With the release of v1.3.0 you can change this using the `sortFacetValuesBy` property of the [`faceting` index settings](/reference/api/settings#faceting). This will allow you to sort facet by value count i.e. the genre with the highest number of results shows first.
+
+Done by @Kerollmops in (#3612)
+
+### Performance improvements
+
+* Use `RoaringBitmap::deserialize_unchecked_from` to reduce the deserialization time (#3788) @Kerollmops 
+* Remove the `docid_word_positions` database (#3819) @loiclec
```

Comment:
You can put it in `Other improvements` sub section

What does the second point? only performance improvement? I would say also reduce the size of the DB

Reply:
Yes I confirm, [Loic's PR](https://github.com/meilisearch/meilisearch/pull/3819) is linked to [this issue](https://github.com/meilisearch/meilisearch/issues/3816)